source/scss/components/_rating-stars.scss

```diff
@@ -1,42 +1,30 @@
 // HOTEL CATEGORY IMAGE ELEMENTS (previously known as Stars)
 
 .mot_stars {
-  display: flex;
-  padding-top: 2px;
+  display: inline-flex;
   white-space: nowrap;
-  width: 72px;
 
   .rating-star:first-child {
     margin-left: 0;
   }
 }
 
 .rating-star {
-  background-color: $orange;
-  border: 1px solid $orange;
+  background-color: $font-color;
+  border: 1px solid $font-color;
   display: inline-block;
-  height: 12px;
-  margin-left: 3px;
+  height: $p-font-size / 2;
+  margin-left: 2px;
   position: relative;
-  width: 12px;
+  top: -($p-font-size / 2);
+  width: $p-font-size / 2;
 
   &.rating-star-empty {
     background-color: transparent;
   }
 
   &.rating-star-half-empty {
-    background-color: transparent;
+    background: linear-gradient(-45deg, transparent 50% , $font-color 50%);
```

Comment:
Will this only value work on all browsers? We should use an online gradient generator like this, for example: https://www.colorzilla.com/gradient-editor/

Reply:
Ankush can have a look

Reply:
[So Ankush had a look at it.](https://jira.fti-group.com/browse/ETS-2009?focusedCommentId=104962&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-104962)
Resolution is, that we keep this more simpler coding, because it not dependent on the element's size.

The outcome is, that is can create some "Blitzer", tiny whitespaces between border and filled background. This is due to the calculation of fractional pixels in ie11 and edge combined with low resolution screens. This also only occurs, when the squares are tiny and it's barely visible due to its tininess.